source/README.adoc

```diff
@@ -0,0 +1,171 @@
+
+:toc:
+:toc-placement!:
+= RabbitMQ Source
+
+The RabbitMQ source translates messages on a RabbitMQ exchange to CloudEvents,
+which can then be used with Knative Eventing over HTTP. The source can bind to
+an existing RabbitMQ exchange, or create a new exchange if required.
```

Comment:
```suggestion
The RabbitMQ source translates messages on a RabbitMQ exchange to CloudEvents,
which can then be used with Knative Eventing over HTTP. The RabbitMQ source can bind to
an existing RabbitMQ exchange, or create a new exchange if required.
```

Reply:
Can we clarify what's meant here by "can then be used with Knative Eventing over HTTP" - used with how?

Reply:
Added more detail linking to the eventing Addressable concept.